addons/mass_mailing/views/utm_campaign_views.xml

```diff
@@ -25,56 +26,31 @@
                     groups="mass_mailing.group_mass_mailing_user">
                     <field name="mailing_mail_ids" readonly="1" nolabel="1">
                         <tree>
-                            <field name="subject"/>
-                            <field name="sent_date"/>
-                            <field name="state"/>
-                            <field name="delivered"/>
-                            <field name="opened"/>
-                            <field name="replied"/>
-                            <field name="bounced"/>
+                            <field name="calendar_date" string="Date"/>
+                            <field name="subject" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
+                            <field name="mailing_model_id" string="Recipients" optional="hide"/>
+                            <field name="user_id" widget="many2one_avatar_user"/>
+                            <field name="ab_testing_enabled" string="A/B Test" groups="mass_mailing.group_mass_mailing_campaign" attrs="{'column_invisible': [('parent.ab_testing_mailings_count', '=', 0)]}"/>
+                            <field name="campaign_id" string="Campaign" groups="mass_mailing.group_mass_mailing_campaign" optional="hide"/>
+                            <field name="received_ratio" string="Delivered (%)"/>
+                            <field name="opened_ratio" string="Opened (%)"/>
+                            <field name="bounced_ratio" string="Bounced (%)" optional="hide"/>
+                            <field name="clicks_ratio" string="Clicked (%)"/>
+                            <field name="replied_ratio" string="Replied (%)"/>
+                            <field name="state" decoration-info="state in ('draft', 'in_queue')" decoration-success="state in ('sending', 'done')" widget="badge"/>
                             <button name="action_duplicate" type="object" string="Duplicate"/>
                         </tree>
                     </field>
-                    <div class="o_utm_campaign_mass_mailing_substats d-flex justify-content-end align-items-center">
```

Comment:
Note that we could also remove css linked to the ``o_utm_campaign_mass_mailing_substats`` class, while removing this. Seems not used anywhere else (see in mass_mailing.scss and mass_mailing_mobile.scss)